e2e/tests/init_test.go

```diff
@@ -243,15 +254,30 @@ func (s *suite) TheSessionForApplicationAndServiceContainsTheSupplier(appName st
 }
 
 func (s *suite) TheApplicationSendsTheSupplierARequestForServiceWithData(appName, supplierName, serviceId, requestData string) {
-	res, err := s.pocketd.RunCurl("", serviceId, requestData)
+	// TODO_TECHDEBT(#179): Once relayminer and appgateserver are running in tilt,
+	// use their respective in-tilt hostnames and run E2E tests in tilt. This
+	// should match the on-chain advertised endpoint for the service with the
+	// given serviceId.
+	res, err := s.pocketd.RunCurl("http://localhost:42069", serviceId, requestData)
 	if err != nil {
 		s.Fatalf("error sending relay request from app %s to supplier %s for service %s: %v", appName, supplierName, serviceId, err)
 	}
-	fmt.Println("OLSH Res", res.Stdout)
+
+	relayKey := getRelayKey(appName, supplierName)
+	s.scenarioState[relayKey] = res.Stdout
+}
+
+// TODO_IN_THIS_COMMIT: move
+func getRelayKey(appName, supplierName string) string {
```

Comment:
1. Consider renaming to `relayReferenceKey` + godoc comment.
2. `TODO_IMPROVE` to use (`sessionId`, `supplierName`) since those are the two values used to create the primary composite key on-chain to uniquely distinguish relays

Reply:
**TL;DR, adding.**

I think we would need to alter the test suite design a bit if we really need to distinguish between individual relays in this E2E suite as it would also require tracking an index (or some unique field) on the relays themselves. I was assuming that we won't be needing that level of discrimination, at least not in this suite and/or any time soon. My focus was to close the E2E relay test loop.